app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] a post has many comments and has many likes, kindly include those relations